app.js

```diff
@@ -48,3 +47,54 @@ $('.nav-ul li').on('click', function(){
     $('#contact-page').fadeIn(350);
   }
 });
+
+Project.initProjectPage = function (){
+  projects.forEach(function(myNewProjectObject){
+    console.log(myNewProjectObject, 'myNewProjectObject');
+    $('#projects').append(myNewProjectObject.toHtml());
+  });
+}
+
+Project.loadAll = function(rawData) {
+
+  // Keeping the commented out code below for future reference
```

Comment:
it's fine to have some comments, but it makes it hard to read when there are so many as well as commented-out code. You might want to clean-up a bit to make everything more readable.